### PR TITLE
0.11.4 release notes

### PR DESCRIPTION
**Pull request by dos65**
minor release to fix issue with `cs install metals` https://github.com/scalameta/metals/pull/3872

### CODE REVIEW

File: website/blog/2022-04-27-aluminium.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.11.4 - Aluminium
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.11.4, which
```
Comment (tanishiking) [inline website/blog/2022-04-27-aluminium.md:8]: Something like this?

```suggestion
We're happy to announce the release of Metals v0.11.4, which includes the hotfix of the issue with `cs install metals`. https://github.com/coursier/coursier/issues/2406
```

Comment (kpodsiad) [inline website/blog/2022-04-27-aluminium.md:8]: Shall we leave a few words about it?
```suggestion
We're happy to announce the release of Metals v0.11.4, which fixes the issue with one of the dependencies not being available on maven central.
```

**Review by kpodsiad — APPROVED** [review]
Thanks @dos65!

File: website/blog/2022-04-27-aluminium.md (hunk 1 above)
Comment (dos65) [reply website/blog/2022-04-27-aluminium.md:8]: oh)
@kpodsiad I will take this one

**Review by tanishiking — APPROVED** [review]

**Pull request merged**